src/mesh/RadioLibRF95.cpp

```diff
@@ -25,7 +25,8 @@ int16_t RadioLibRF95::begin(float freq, float bw, uint8_t sf, uint8_t cr, uint8_
     RADIOLIB_ASSERT(state);
 
 #ifdef RF95_TCXO
-    state = _mod->SPIsetRegValue(RADIOLIB_SX127X_REG_TCXO, 0x10 | _mod->SPIgetRegValue(RADIOLIB_SX127X_REG_TCXO));
+    Module *_mod = this->getMod();
+    state = _mod->SPIsetRegValue(RADIOLIB_SX127X_REG_TCXO, 0x10 | _mod->SPIgetRegValue(SX127X_REG_TCXO));
```

Comment:
this doesn't look right.  should the argument to setReg be the same as getReg?

Reply:
> this doesn't look right. should the argument to setReg be the same as getReg?

I will have to double check this against my code on my other PC later